ansible/ha-meta.yml

```diff
@@ -0,0 +1,230 @@
+# This playbook gathers meta data facts for SAP/HANA checker.
+
+- hosts: all
+
+  tasks:
+    - name: include load_facts
+      import_role:
+        name: load_facts
+      delegate_to: localhost
+      run_once: yes
+
+    - name: include 1.1.1 meta
+      import_role:
+        name: '1.1.1'
+        tasks_from: meta
+      tags:
```

Comment:
Together with the other propose change, I would remove the `tags` and `run_once`. They don't need to be there

Reply:
OK.